rsp-starter/templates/ingress.yaml

```diff
@@ -27,7 +27,6 @@ spec:
         paths:
           - path: {{ .Values.ingress.path }}
```

Comment:
Please consider changing this to 
```
- path: {{ default "/<CHARTNAME>" .Values.ingress.path }}
```
so that by default the service is available at the obvious location. That goes well with the default pathType.